pkgs/games/r2modman/default.nix

```diff
@@ -0,0 +1,77 @@
+{ lib
+, mkYarnPackage
+, fetchFromGitHub
+, gitUpdater
+, makeWrapper
+, makeDesktopItem
+, electron
```

Comment:
Should we pin the electron major version?

Reply:
I don't think there's any need to unless the application is broken in current Electron.

Reply:
Makes sense, though it could be helpful to make sure future electron updates don't break anything.

Reply:
Yeah leaving it as just `electron` until it breaks is more desirable in nixpkgs

Electron releases move pretty quickly and it'll easily get left behind otherwise. Usually if an app works with a newer Electron, it is better for it to (since they also have to fix bugs so often)